node-webservice/react-client/src/Components/BehaviourBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import '../CSS/BehaviourBar.css'
+
+class BehaviourBar extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div className="rating">
+                <form className="rating-form">
+                    <label>
+                        <input type="radio" name="rating" className="super-happy" id="super-happy" value="super-happy" onClick = {() =>this.props.clicked("Defend")} />
```

Comment:
Looks good, we should just be aware that using ids in a react component can make less reusable, although I cant imagine why we would need multiple BehaviorBars at the same time